### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by toxaus**
add some fixes for existing converters.
Also add new converter PB UA.

### CODE REVIEW

File: CurrencyConverter/Converters/PrivatBankUA.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace CurrencyConverter\Converters;
+
+use \CurrencyConverter\Converter;
+use \CurrencyConverter\ConverterException;
+
+class PrivatBankUA extends Converter
+{
+    /* Different request formats */
```
Comment (zhikiri) [inline CurrencyConverter/Converters/PrivatBankUA.php]: @toxaus Don't really agree with this one line comment style "/*", prefer "//"

File: CurrencyConverter/Converters/Yahoo.php [hunk 2]
```diff
@@ -11,7 +11,11 @@ class Yahoo extends Converter
     public function rate($from, $to)
     {
         $response = $this->send("select * from yahoo.finance.xchange where pair in (\"{$from}{$to}\")");
-        if (!is_array($response) || !isset($response["query"]["results"]["rate"]["Rate"])) {
+        if (!is_array($response) || !isset($response["query"]["results"]["rate"]["Rate"])
+           || (!isset($response["query"]["results"]["rate"]["Rate"])
```
Comment (zhikiri) [inline CurrencyConverter/Converters/Yahoo.php]: @toxaus seems there is the same check for this: <code>!isset($response["query"]["results"]["rate"]["Rate"])</code>

File: CurrencyConverter/Converters/Yahoo.php [hunk 3]
```diff
@@ -21,14 +25,15 @@ public function rate($from, $to)
...
 
-        $response = file_get_contents(sprintf(
-            "https://query.yahooapis.com/v1/public/yql?%s",
-            http_build_query([
-                "q" => $query,
-                "format" => $format,
-                "env" => "store://datatables.org/alltableswithkeys",
-            ])
-        ));
+        $url      = sprintf(
```
Comment (zhikiri) [inline CurrencyConverter/Converters/Yahoo.php]: @toxaus Don't see utility of this, as for me current version looks more fancy )

Comment (toxaus) [inline CurrencyConverter/Converters/Yahoo.php]: For debugging of problems, its very usefull. You always can see requested URL, no?

File: CurrencyConverter/Converters/Yahoo.php (hunk 2 above)
Comment (toxaus) [inline CurrencyConverter/Converters/Yahoo.php]: agree, seems we can skip just this 1 condition

**Pull request merged**